chartfx-dataset/src/test/java/de/gsi/dataset/spi/EditableDataSetTests.java

```diff
@@ -22,12 +21,12 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(EditableDataSetTests.class);
     protected static final int N_POINTS_INSERT_FRONT = 3;
     protected static final int N_POINTS_ADD_BACK = 5;
-    protected static final double[] testCoordinates = { 1.0, 2.0 };
-    protected static final double[] errorCoordinates = { 0.1, 0.2, 0.3, 0.4, 0.5, 0.6 };
+    protected static final double[] testCoordinates = {1.0, 2.0};
+    protected static final double[] errorCoordinates = {0.1, 0.2, 0.3, 0.4, 0.5, 0.6};
 
     @Test
-    public void testCommonDataSetImplementations() {
-
+    public void
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=4730376)